Formula/httprs.rb

```diff
@@ -0,0 +1,21 @@
+class Httprs < Formula
+  desc "Simple and configurable command-line HTTP server"
+  homepage "https://github.com/http-server-rs"
+  url "https://github.com/http-server-rs/http-server.git",
+       tag:      "v0.8.5",
+       revision: "2374c3281e53f7e98c17079728e6bf70b8c8241d"
+  license any_of: ["Apache-2.0", "MIT"]
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--path", "./"
+    mv "~/.cargo/bin/http-server", HOMEBREW_PREFIX/"bin/httprs"
+  end
+
+  test do
+    assert_equal which("httprs"), opt_bin/"httprs"
+    assert_predicate HOMEBREW_PREFIX/"bin/httprs", :exist?, "httprs is not pesent"
+    assert_predicate HOMEBREW_PREFIX/"bin/httprs", :executable?, "httprs is not executable"
```

Comment:
Can you propose a more substantial test than checking the version, calling for help, or checking the existence of binaries? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula